serverservice/consts.go

```diff
@@ -1,12 +1,30 @@
 package serverservice
 
 const (
-	// Serverservice attribute namespace for device vendor, model, serial attributes
+	// Serverservice attribute namespace for device vendor, model, serial attributes.
 	ServerAttributeNSVendor = "sh.hollow.alloy.server_vendor_attributes"
 
-	// Serverservice attribute namespace for the BMC address
+	// Serverservice attribute namespace for the BMC address.
 	ServerAttributeNSBmcAddress = "sh.hollow.bmc_info"
 
-	// Serverservice attribute namespace for firmware set labels
+	// Serverservice attribute namespace for firmware set labels.
 	FirmwareAttributeNSFirmwareSetLabels = "sh.hollow.firmware_set.labels"
+
+	// Serverservice attribute namespace for inband firmware information.
+	FirmwareVersionInbandNS = "sh.hollow.alloy.inband.firmware"
+
+	// Serverservice attribute namespace for outofband firmware information.
+	FirmwareVersionOutofbandNS = "sh.hollow.alloy.outofband.firmware"
+
+	// Serverservice attribute for installed firmware.
+	FirmwareVersionNSInstalledAttribute = "firmware.installed"
```

Comment:
How does this differ from `sh.hollow.alloy.inband.firmware`? Is this the combined attribute? If so, should we follow the established naming conventions?

Reply:
This const identifies the field within `data` to look for the installed firmware version `firmware->installed` 
```json
   "versioned_attributes": [
        {
          "namespace": "sh.hollow.alloy.inband.firmware",
          "data": {
            "firmware": {
              "installed": "20.5.13"
            }
          },
```

Reply:
Can we bury this nuance either in FleetDB or the health-data controller? It doesn't feel like this is a facet that is (or should be) truly shared between multiple components.

Reply:
I missed replying here, 

This `FirmwareVersionNSInstalledAttribute` value is used to build the HSS query to list installed firmware `?attr=sh.hollow.alloy.outofband.firmware~firmware.installed~eq~20.5.13` and so it makes sense to have this shared.

Reply:
Bluntly, I don't really like this. It's an artifact of how HSS stores firmware data (which we agree is not awesome). It's OK for now, but I'd like to address this in the FleetDB schema re-work.